homopolymer_fit/Ising_fitter_homopolymer.ipynb

```diff
@@ -28,17 +26,21 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "import numpy as np\n",
-    "import ntpath   \n",
+    "from __future__ import division\n",
+    "import numpy as np  \n",
     "import glob     \n",
     "import csv\n",
     "import json\n",
+    "import os\n",
     "import time\n",
     "import pandas as pd\n",
+    "import sympy as sp\n",
+    "import matplotlib.pyplot as plt\n",
+    "from matplotlib.backends.backend_pdf import PdfPages\n",
+    "import lmfit\n",
+    "import math\n",
     "\n",
-    "path = '/Users/dougbarrick/OneDrive - Johns Hopkins/Manuscripts/Ising_program/\\\n",
-    "Scripts/scripts_vanilla_NRC/cANK_scripts/cANK_jupyter_scripts/'\n",
```

Comment:
removed hardcoded path and replaced with one dynamic to the user system

Reply:
We will add a set of instructions to the README.md to let users know how to spin up these environments and where to run the jupyter notebook command for making this work.